velox/exec/Task.h

```diff
@@ -33,6 +33,11 @@ class HashJoinBridge;
 class NestedLoopJoinBridge;
 class Task : public std::enable_shared_from_this<Task> {
  public:
+  /// The task execution state which is also used for reporting query
+  /// completion.
+  enum class State { kRunning, kFinished, kCanceled, kAborted, kFailed };
+  static std::string stateString(State state);
```

Comment:
This refactoring is nice, but makes it hard to see material changes. Would it be possible to split this PR into pure refactoring and actual fix?